firefox-ios/RustFxA/FirefoxAccountSignInViewController.swift

```diff
@@ -238,7 +240,7 @@ class FirefoxAccountSignInViewController: UIViewController, Themeable {
         scanButton.applyTheme(theme: theme)
         emailButton.applyTheme(theme: theme)
 
-        scanButton.configuration?.image = signinSyncQRImage?.withRenderingMode(.alwaysTemplate)
+        scanButton.setImage(signinSyncQRImage?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
While this certainly works I don't think we should set the image this way. The button is using a button configuration and the image should be set the same way through the configuration. We should rather figure out why the button doesn't resize correctly when setting the image using the button configuration and fix that issue.

Reply:
@DanielDervishi I think you forgot to change this code back to use the configuration.

Reply:
@thatswinnie Whoops, totally though I changed that back. Will fix ASAP